src/api/comment/content-types/comment/schema.json

```diff
@@ -29,6 +29,12 @@
       "relation": "manyToOne",
       "target": "plugin::users-permissions.user",
       "inversedBy": "comments"
+    },
```

Comment:
This might be added, as you may had an interim version of the database. It is suggested to run `npm run init:db` to recreate development database. For now you can just remove this and contentTypes.d.ts additions

Reply:
Are there a scenario that this and `contentTypes.d.ts` are necessary ?

Reply:
I runned `npm run init:db`, and it didn't removed the fiels, I had to do it manually.

Reply:
`contentTypes.d.ts` is needed, and your removal of Point from it, is also needed to be committed, but you had an extra bit added on `authorId`. Running `npm run init:db` will not fix `contentTypes.d.ts` or any other files, but I advise to run it before you start working on a new task, so that you have the latest database state and avoid picking unrelated changes from you local DB.

Reply:
> `contentTypes.d.ts` is needed, and your removal of Point from it, is also needed to be committed, but you had an extra bit added on `authorId`.

The error was because I added `authorId` at `src/api/comment/content-types/comment/schema.json`.
rigth ?

Reply:
> Running `npm run init:db` will not fix `contentTypes.d.ts` or any other files, but I advise to run it before you start working on a new task, so that you have the latest database state and avoid picking unrelated changes from you local DB.

OK, thank you I will take notes about this. Let me know if that is rigth:

- Start work in a new task:
  - Pull main.
  - Fetch branchs.
  - Run `npm run init:db` to have the latest database state and avoid picking unrelated changes to the local DB.

Looks good ?
Enjoy this opportunity, another tip when start working in a new task ?